frontend/public/components/utils/resource-icon.tsx

```diff
@@ -32,7 +32,6 @@ export const ResourceIcon: React.SFC<ResourceIconProps> = ({
 
   const rendered = (
     <>
-      <span className="sr-only">{kindStr}</span>
```

Comment:
I enjoyed reading the complete PR, but then I found this. Any reasons you have removed this?

Reply:
@jerolimov  I tried to fix this issue here https://github.com/openshift/console/pull/12400#issuecomment-1374859149

the "sr-only" class in the ResourceIcon is adding "position:absolute" is causing the above issue

I fixed it and added "position:relative" in the ContainerField component